presto-docs/src/main/sphinx/release/release-324.rst

```diff
@@ -20,6 +20,11 @@ Security Changes
 * Perform access control checks when displaying table or view definitions
   with ``SHOW CREATE``. (:issue:`1517`)
 
+Web UI Changes
```

Comment:
Shouldn't this comes under release-325 ?

Reply:
Oops, yeah, I updated the wrong file. Good catch